File: test/fixtures/node-axios/j_post_data_wo_verb.js
const axios = require('axios');

const response = await axios.post('http://localhost:28139/post', {
    headers: {
        'Content-Type': 'application/x-www-form-urlencoded'
    },
    data: 'data1=data1&data2=data2&data3=data3'
});

File: test/fixtures/ruby-httparty/j_post_data_wo_verb.rb
require 'httparty'

url = 'http://localhost:28139/post'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = {
  'data1' => 'data1',
  'data2' => 'data2',
  'data3' => 'data3'
}
res = HTTParty.post(url, headers: headers, body: body)
